clients/geth/specular/rollup/services/validator/validator.go

```diff
@@ -180,7 +192,12 @@ func (v *Validator) getNextAssertionAttrs(ctx context.Context) (assertionAttribu
 	if err != nil {
 		return assertionAttributes{}, fmt.Errorf("failed to get finalized assertion attrs: %w", err)
 	}
-	return assertionAttributes{header.Number.Uint64(), header.Root}, nil
+
+	inboxSize, err := v.sequencerInbox.GetInboxSize(nil)
```

Comment:
this is incorrect, you need to get the inbox size corresponding to the header, or the header corresponding to the inbox size

Reply:
rolled back this change, inboxSize is now measured in terms of # blocks